html/pages/services.inc.php

```diff
@@ -141,9 +138,14 @@
                     } else {
                         $sql_param[0] = $device_id;
                     }
-                    $head = true;
 
-                    foreach (dbFetchRows("SELECT * FROM `services` WHERE `device_id` = ? $where ORDER BY service_type", $sql_param) as $service) {
+                    $header = true;
+                    $footer = false;
+
+                    $svc = 0;
```

Comment:
I'd like to see better variable names that describe what data they contain. Same for the two below.